src/lib/dvf/delete-authenticated.js

```diff
@@ -0,0 +1,9 @@
+const addAuthHeadersOrData = require('./addAuthHeadersOrData')
+const deleteGeneric = require('./delete-generic')
+
+module.exports = async (dvf, endpoint, nonce, signature, data = {}) => {
+  const { headers, data: json } = await addAuthHeadersOrData(
```

Comment:
Any new endpoints should only ever be authenticated via headers so let’s remove the json bit here (or better still, throw if json is defined and non-empty)

Reply:
since this is a new function there should be no legacy places making use of data, so I decided to just remove it